src/quo/components/links/internal_link_card/community/view.cljs

```diff
@@ -0,0 +1,96 @@
+(ns quo.components.links.internal-link-card.community.view
+  (:require
+    [quo.components.community.community-stat.view :as community-stat]
+    [quo.components.links.internal-link-card.community.style :as status-link-previews-style]
+    [quo.components.links.link-preview.style :as style]
+    [quo.components.markdown.text :as text]
+    [quo.theme :as theme]
+    [react-native.core :as rn]))
+
+(defn- description-comp
+  [description members-count active-members-count]
+  [rn/view
+   [text/text
+    {:size                :paragraph-2
+     :number-of-lines     3
+     :accessibility-label :description} description]
+   [rn/view {:flex-direction :row :margin-top 12}
+    [community-stat/view
+     {:value               members-count
+      :icon                :i/members
+      :accessibility-label :members-count
+      :style               {:margin-right 12}}]
+    [community-stat/view
+     {:value               (or active-members-count 0)
+      :icon                :i/active-members
+      :accessibility-label :active-members-count}]]])
+
+(defn- title-comp
+  [title]
+  [text/text
+   {:size                :paragraph-1
+    :number-of-lines     1
+    :weight              :semi-bold
+    :style               style/title
+    :accessibility-label :title}
+   title])
+
+(defn- thumbnail-comp
+  [thumbnail]
+  [rn/image
+   {:style               status-link-previews-style/thumbnail
+    :source              (if (string? thumbnail)
```

Comment:
This workaround for images is no longer necessary. Some time ago I changed `rn/image` to do the work for us.

https://github.com/status-im/status-mobile/blob/1385e27cf46445a62a512a864387f765112b8421/src/react_native/core.cljs#L28